src/dom/matrix-dom.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Class for dealing with CSS Transform matrices.
+ * Supports relative updates being made to a matrix from multiple pieces of
+ * code without them over-riding each other (unless explicitly desired).
+ */
+import { dom } from './dom';
+
+type Numeric = number|string; // Number that could be represented as string
```

Comment:
I've been thinking about this some more, and since typescript is strict about types, I think it's better that only number values are accepted below and that users of this class (including `parseFromString`) are required to pass in the correct type.

Otherwise passing an invalid value like `'a'` will produce `NaN` and cause problems.

Reply:
Done. Added a check in parseFromString to make sure the values grabbed are appropriate length and not-NaN, throwing an error if used with incorrect values.